src/components/FilterBar/FilterBar.jsx

```diff
@@ -1,96 +1,218 @@
+import { useFilterContext } from "../../context/filter-context";
 import "./FilterBar.css";
 const FilterBar = () => {
+  const { productListState, productListDispatch } = useFilterContext();
   return (
     <section>
-      <div class="all-contain">
-        <div class="filters">
-          Filters <span className="clear-all">Clear</span>
+      <div className="all-contain">
+        <div className="filters">
+          Filters{" "}
+          <span
+            className="clear-all"
+            onClick={() => productListDispatch({ type: "" })}
+          >
+            Clear
+          </span>
         </div>
 
-        <div class="main-category">Sort By</div>
-        <li class="category-list">
-          <label class="category-name">
-            <input type="radio" name="sort" id="" /> Price - Low to High
+        <div className="main-category">Sort By</div>
+        <li className="category-list">
+          <label className="category-name">
```

Comment:
refer to the element for which this label is being used

Reply:
yup will fix that